src/pages/List.js

```diff
@@ -1,42 +1,15 @@
-import React, { useState, useEffect } from 'react';
-import { collection, onSnapshot, query } from 'firebase/firestore';
-import { db } from '../lib/firebase';
-import { NavLink } from 'react-router-dom';
+import React from 'react';
+import ItemList from '../components/ItemList';
 import '../components/AddItemForm.css';
+import '../components/Item.css';
```

Comment:
Same note as before:  These 2 imports aren't actually importing anything and can be removed